packages/ui-kit-components/src/components/Graph/graph.tsx

```diff
@@ -126,6 +126,7 @@ export interface GraphProps {
   labels: string[];
   data: PartialNumberType[][];
   previousData?: number;
+  currentData?: number;
```

Comment:
The current price line is missing the label that goes on the left hand side of the graph.

Reply:
![Screen Shot 2022-12-13 at 3 55 43 PM](https://user-images.githubusercontent.com/106985561/207442233-bd605cdc-41c3-472f-807c-68cfece36277.png)
![Screen Shot 2022-12-13 at 3 56 30 PM](https://user-images.githubusercontent.com/106985561/207442234-08d63990-da9d-44bc-9582-3eb458121010.png)
![Screen Shot 2022-12-13 at 3 56 41 PM](https://user-images.githubusercontent.com/106985561/207442235-6a8e3333-7020-4542-9316-6c02053f4f9d.png)
![Screen Shot 2022-12-13 at 3 56 57 PM](https://user-images.githubusercontent.com/106985561/207442238-1ed6a053-c416-46e0-aee6-dc8ac0ada469.png)